botx/bot/bot.py

```diff
@@ -1214,6 +1218,44 @@ async def get_sticker(
 
         return botx_api_sticker.to_domain()
 
+    async def edit_sticker_pack(
+        self,
+        *,
+        bot_id: UUID,
+        sticker_pack_id: UUID,
+        name: str,
+        preview: UUID,
+        sticker_order_id: List[UUID],
+    ) -> StickerPack:
+        """Edit Sticker pack.
+
+        Args:
+            bot_id: Bot which should perform the request.
+            sticker_pack_id: Sticker pack id.
+            name: Sticker pack name.
+            preview: Sticker from the set selected as a preview.
+            sticker_order_id: List Sticker IDs in set in order they are displayed.
```

Comment:
`Sticker IDs in order they are displayed`

Reply:
Thank you. Fix